fs_storage/rooted_dir_file_system.py

```diff
@@ -20,18 +20,20 @@ class RootedDirFileSystem(DirFileSystem):
     """
 
     def _join(self, path):
-        path = super()._join(path)
-        # Ensure that the path is a subpath of the root path by resolving
-        # any relative paths.
```

Comment:
@hnavarro-kernet Thank you for the fix. Nevertheless can you keep the first 2 comment lines